packages/react/src/progress-indicator/ProgressIndicatorItemLink.tsx

```diff
@@ -0,0 +1,47 @@
+import { LinkProps } from '../core';
+import { TextLink } from '../text-link';
+import { ProgressIndicatorItem } from './ProgressIndicatorItem';
+import {
+	ProgressIndicatorBackground,
+	ProgressIndicatorItemStatus,
+} from './utils';
+
+export type ProgressIndicatorItemLinkProps = LinkProps & {
+	/** Determines what style the item displays as.
+	 *
+	 * `blocked` = This step is not yet available for the user to start.
+	 *
+	 * `doing` = **(deprecated)** This step has been started and the user is actively working on this step (Use `started` with `isActive: true` instead).
+	 *
+	 * `done` = This step is complete. The user has submitted valid data and it has been verified (where necessary).
+	 *
+	 * `error` = This step has an error that requires attention.
+	 *
+	 * `saved` = This step has valid saved data that can still be changed.
+	 *
+	 * `started` = This step has been started and still needs to be completed.
+	 *
+	 * `todo` = This step is available for the user to start.
+	 */
+	status: ProgressIndicatorItemStatus;
```

Comment:
Could alphabetise again

Reply:
Ahh interesting that you raise this. Something I'm doing unconsciously is grouped ordering + alphabetising. So I put the required props group first and alphabetise those, then the optional props and alphabetise them.

This adds significance to the required props by putting them at the top which is something particularly useful in components with large numbers of props. It also gives us props tables in docs that have the required props at the top.

![image](https://github.com/agriculturegovau/agds-next/assets/43688949/1bfb5f30-13f8-46cf-a8ba-f047da73cbaa)

Reply:
Fair enough. Probably can't be automated in future, but fair enough